src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/SymbolManagerBase.cs

```diff
@@ -337,16 +331,18 @@ public bool Equals(TypeArrayKey other)
                 if (other._types == _types) return true;
                 if (other._types.Length != _types.Length) return false;
                 if (other._hashCode != _hashCode) return false;
-                for (int i = 0, n = _types.Length; i < n; i++)
+                for (int i = 0; i < _types.Length; i++)
                 {
                     if (!_types[i].Equals(other._types[i]))
                         return false;
                 }
                 return true;
             }
 
+            [ExcludeFromCodeCoverage] // Typed overload should always be the method called.
```

Comment:
Seems a shame to ship 100's of these attributes to customers but I guess we already do it;.